tendrl-dashboard.spec

```diff
@@ -8,16 +8,57 @@ Group:   Applications/System
 Source0: %{name}-%{version}.tar.gz
 URL: https://github.com/Tendrl/dashboard
 
-Requires: tendrl-api
+BuildRequires: npm
+BuildRequires: nodejs-packaging
+BuildRequires: fontconfig
 
 %description
 Contains the JavaScript GUI content for the Tendrl front-end components
 (dashboard, login screens, administration screens)
 
 %prep
-%autosetup
+%setup -q -n %{name}-%{version}
+%nodejs_fixdep foomodule
 
 %build
+npm install gulp
```

Comment:
Just a heads' up that `npm install` will not work in Koji. For reproducibility's sake, all artifacts must be present in dist-git or the lookaside cache, and in fact kojid is is firewalled from downloading artifacts over the internet to preserve reproducibility.

See the "Disable domain name resolution" section of the [mock wiki page](https://fedoraproject.org/wiki/Using_Mock_to_test_package_builds#Disable_domain_name_resolution) for a way to simulate this apart from Koji.